modules/phase_field/include/kernels/NestKKSSplitCHCRes.h

```diff
@@ -0,0 +1,85 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#pragma once
+
+#include "DerivativeMaterialInterface.h"
+#include "JvarMapInterface.h"
+#include "Kernel.h"
+
+// Forward Declarations
+
```

Comment:
```suggestion
```

Reply:
Should we remove the `// Forward Declarations` for the kernels I added or just the one you commented on?